opentelemetry-quickstart/otel-collector-config.yaml

```diff
@@ -0,0 +1,29 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: otel-collector:4317
+  otlp/2:
+    protocols:
+      grpc:
+        endpoint: otel-collector:55680
+
+exporters:
+  jaeger:
+    endpoint: jaeger-all-in-one:14250
+    insecure: true
```

Comment:
```suggestion
    tls:
      insecure: true
```

Reply:
Yes, this changed a few versions ago. This is the new configuration.